src/app/modules/registration/components/edit-images/edit-images.component.html

```diff
@@ -40,11 +40,16 @@
   <ion-label>{{ title | translate }}</ion-label>
 </ion-item>
 
-<div class="file-drop" *ngIf="!isHybrid">
-  <ngx-file-drop dropZoneLabel="Drop files here" [accept]="accept" dropZoneClassName="drop-zone" (onFileDrop)="dropped($event)">
+<ion-item class="file-drop" *ngIf="!isHybrid">
+  <ngx-file-drop style="width:100%;" dropZoneLabel="Drop files here" [accept]="accept" dropZoneClassName="drop-zone"
+    (onFileDrop)="dropped($event)">
     <ng-template ngx-file-drop-content-tmp let-openFileSelector="openFileSelector">
-      <input type="file" />
-      <button type="button" (click)="openFileSelector()"></button>
+      <button type="button" (click)="openFileSelector()">
+        <ion-item [ngClass]="aboutToDrop && 'grey'" (dragenter)="aboutToDrop = true" (dragleave)="aboutToDrop=false">
```

Comment:
Jeg tror du kan bruke `[ngClass]="{'grey': aboutToDrop}"` for å unngå at det legges på en klasse `"false"` som ikke finnes når `aboutToDrop` er `false`?